mods/ts/rules/misc.yaml

```diff
@@ -32,6 +32,8 @@ harv.colorpicker:
 	Inherits: HARV
 	-Buildable:
 	-MapEditorData:
+	RenderSprites:
+		Image: harv
```

Comment:
Why does the colorpicker actor need this now? Just because of linting? The `Buildable` trait was removed on this actor.

Reply:
Ah no, the change makes sense to me now, thanks.